policy/modules/system/init.te

```diff
@@ -294,6 +300,16 @@ ifdef(`init_systemd',`
 	# /memfd:systemd-state
 	fs_tmpfs_filetrans(init_t, init_runtime_t, file)
 
+	allow init_t init_var_lib_t:dir manage_dir_perms;
+	allow init_t init_var_lib_t:file manage_file_perms;
+	allow init_t init_var_lib_t:lnk_file manage_lnk_file_perms;
```

Comment:
this is probably for `/var/lib/systemd/timers`?

the lnk file is a bit questionable, but in theory I think it is a possible scenario for example `StateDirectory=systemd/timesync` in that case systemd will create the `/var/lib/systemd/timesync` symlink that points to `/var/lib/private/systemd/timesync`

Reply:
I only really see these accesses on `/var/lib/systemd/timers`, yes, though I'm not sure if they might occur elsewhere there. In my case I only ever observed systemd trying to `setattr` on the timer files in order to update their timestamps.